server/go/api_apis_service_impl.go

```diff
@@ -35,14 +35,14 @@ func NewApisApiService(kuskClient kusk.Client) ApisApiServicer {
 
 // GetApis - Get a list of APIs
 func (s *ApisApiService) GetApis(ctx context.Context, namespace string, fleetname string, fleetnamespace string) (ImplResponse, error) {
-	apis := &kuskv1.APIList{}
+	var apis *kuskv1.APIList
 	var err error
+
 	if fleetname == "" {
 		apis, err = s.kuskClient.GetApis(namespace)
```

Comment:
```suggestion
		apis, err := s.kuskClient.GetApis(namespace)
```

Reply:
it is an `if` `else` so the variable isn't visible when returning on line 52 where I pass variable apis to the method `convertAPIListCRDtoAPIsModel`